fe/fe-core/src/test/java/com/starrocks/connector/parser/trino/TrinoQueryTest.java

```diff
@@ -703,6 +703,12 @@ public void testSelectCTE() throws Exception {
         sql = "with x0 as (select * from t0) " +
                 "select * from x0 x,t1 y where v1 in (select v2 from x0 z where z.v1 = x.v1)";
         assertPlanContains(sql, "8:NESTLOOP JOIN", "LEFT SEMI JOIN (PARTITIONED)");
+
+        sql = "with C1 as (select * from t0) select * from C1";
```

Comment:
```suggestion
        sql = "with C1 as (select * from t0) select * from c1";
```
How about changing table reference from `C1` to `c1`.

Reply:
ok, I will add this case